src/main/java/org/reaktivity/nukleus/sse/internal/stream/SseServerFactory.java

```diff
@@ -908,26 +909,21 @@ else if (name.equals(HEADER_NAME_METHOD))
                         .payload(sseEvent.buffer(), sseEvent.offset(), sseEvent.sizeof())
                         .build();
 
-                final int networkReplyDebit = sseEvent.sizeof() + networkReplyPadding;
-                if (networkReplyBudget > networkReplyDebit)
+                if (networkSlot == NO_SLOT)
                 {
-                    networkReplyBudget -= networkReplyDebit;
-                    networkReply.accept(data.typeId(), data.buffer(), data.offset(), data.sizeof());
+                    networkSlot = bufferPool.acquire(networkReplyId);
                 }
-                else
-                {
-                    if (networkSlot == NO_SLOT)
-                    {
-                        networkSlot = bufferPool.acquire(networkReplyId);
-                    }
 
-                    if (networkSlot != NO_SLOT)
-                    {
-                        MutableDirectBuffer buffer = bufferPool.buffer(networkSlot);
-                        buffer.putBytes(networkSlotOffset, data.buffer(), data.offset(), data.sizeof());
-                        networkSlotOffset += data.sizeof();
-                    }
+                if (networkSlot != NO_SLOT)
+                {
+                    MutableDirectBuffer buffer = bufferPool.buffer(networkSlot);
+                    buffer.putBytes(networkSlotOffset, data.buffer(), data.offset(), data.sizeof());
+                    networkSlotOffset += data.sizeof();
+                    deferredCredit += data.sizeof();
```

Comment:
Should be `deferredClaim`, not `deferredCredit`, right?
Also `deferredClaim` would be `+= data.reserved()`, agreed?

Reply:
we are adding into buffer only `data.sizeof()` worth of data right not the `data.reserved()`. Wouldn't it be wrong then to claim multiple `reserved`?

Reply:
I was thinking that the amount you claim should match what is credited back later. 

Since the later credit will be based on data reserved, then it would follow that we should be claiming the value of data reserved.